src/Display/Tabs/TabsStyle.ts

```diff
@@ -63,7 +72,8 @@ export const TabsHeader = styled.div`
     }
 
     .horizontal-tab.underlined {
-      margin: 0 10px;
+      margin: ${({ colour }) =>
+        colour === ETabColourVariant.GREY ? '0 15px' : '0 10px'};
```

Comment:
There are a lot of `colour === ETabColourVariant.GREY` conditions, I was thinking if we can have an intuitive variable instead. e.g. `const isNewTab = colour !== ETabColourVariant.BLACK`. Because it is a bit hard to infer why we change the style based on color.